### ISSUE TITLE
interleave_data returns non-sense when not all pores have been assigned to a Geometry

### ISSUE THREAD

**Issue by jgostick** [PMEAL/OpenPNM#638]
If you do:

``` python
pn = op.Network.Cubic(shape=[4,4,4])
geom = op.Geometry.Stick_and_Ball(network=pn, pores=pn.Ps[:-2])
```

Then this array will contain non-sensical values in the last two elements:

``` python
pn['pore.diameter']
```

**Comment by jgostick** [comment]
Options are:
1. issue a warning and then fill missing spaces with NaNs
2. throw an exception of some sort

**Comment by jgostick** [comment]
solved by PR #665
